felipeknorrkuhn/FELIPE_KNORR_KUHN.md

```diff
@@ -0,0 +1,19 @@
+# Automation against humanity
+
+Stress. Anxiety. Anger. Frustration. None of these words should be associated with a shopping experience. Still, when the COVID-19 pandemic started, several supply chains were affected, causing high demand and low stock for electronics, food, and general household goods worldwide. From toilet paper to flour to video games and N95 masks - at traditional brick and mortar stores as well as e-commerce, it was a real challenge to stock up for the months full of unknowns ahead of us.
+
+Talking about videogames more specifically, during the first weeks of March 2020, the demand for a Nintendo Switch fitness game called Ring Fit Adventure was so high in countries already under shelter-in-place orders that it went out of stock globally. Getting out of the door to exercise was not going to be possible until further notice, and people who wanted a copy of the game were out of luck unless they paid two or three times over retail price for one. It took several months for additional units to become available, and only over one year after the pandemic started, some stores marked down the game's price as the interest - and the sense of necessity - faded away.
+
+Fast forward a few months, in November 2020, the Sony PlayStation 5 and the Microsoft Xbox One Series consoles launched, but getting ahold of them was, again, almost as hard as finding water in the desert. Gamers, who were still under the shelter-in-place orders, were desperate to buy their units, but something interesting and unfortunate prevented them from ordering them.
+
+Scalpers identified a good oportunity to profit with these highly anticipated videogame consoles and used specialized bots to secure multiple units before legitimate customers could have a chance to buy one. Some bots were built using popular UI automation libraries well known in the software industry. Others had direct access to APIs provided by retailers to third-party sellers, which enabled them to detect when new stock was available and allowed them to place an order before the items showed up on the storefronts.
```

Comment:
opportunity needs another p

Reply:
consider just putting November as it is just a few months later and 2020 was mentioned previously